### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by sdelamo**

### CODE REVIEW

File: src/main/docs/guide/storage/storageWithStore.adoc [hunk 1]
```diff
@@ -1,9 +1,9 @@
-The following example shows how an equivalent implementation which leverages the ann:microstream.annotation.Store[] annotation to simplify object storage.
+The following example shows how an equivalent implementation which leverages the <<annotations, Micronaut Microstream annotations>> to simplify object storage.
```
Comment (timyates) [inline src/main/docs/guide/storage/storageWithStore.adoc:1]: MicroStream

File: src/main/docs/guide/storage/storageWithStore.adoc [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
 snippet::io.micronaut.microstream.docs.CustomerRepositoryStoreImpl[tags=clazz]
 
 <1> You can also inject an instance of api:microstream.RootProvider[] to easily access the MicroStream Root Instance. If your Micronaut application has only a single MicroStream Instance, you don't need to specify a name qualifier to inject it.
 <2> https://docs.microstream.one/manual/storage/storing-data/index.html[The rule is: "The Object that has been modified has to be stored!"].
 <3> https://docs.microstream.one/manual/storage/storing-data/index.html[To store a newly created object, store the "owner" of the object].
 
-NOTE: ann:microstream.annotation.Store[] only works for synchronous methods. It does not do any logic for methods returning a `Publisher` or a `CompletableFuture`. For those scenarios, use directly the `StorageManager`.
+NOTE: <<annotations, Micronaut Microstream annotations>> only work for synchronous methods. It does not do any logic for methods returning a `Publisher` or a `CompletableFuture`. For those scenarios, use directly the `StorageManager`.
```
Comment (timyates) [inline src/main/docs/guide/storage/storageWithStore.adoc:9]: MicroStream

File: src/main/docs/guide/annotations.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+| It stores the method parameters specified in the annotation value.
+
+| ann:microstream.annotations.StoreReturn[]
+| It stores the method return
+
+| ann:microstream.annotations.StoreRoot[]
+| It stores the root object.
+
+| ann:microstream.annotations.Store[]
+| Micronaut Microstream maps other annotations to this annotation. You will not use this annotation directly.
```
Comment (timyates) [inline src/main/docs/guide/annotations.adoc:18]: MicroStream

**Review by timyates — CHANGES_REQUESTED** [review]
Looks great!  Just a few capitalizations

File: src/main/docs/guide/annotations.adoc (hunk 3 above)
Comment (sdelamo) [reply src/main/docs/guide/annotations.adoc]: resolved https://github.com/micronaut-projects/micronaut-microstream/pull/74/commits/1019bbc521b61fdf7033d91b9283c90780d006e4

File: src/main/docs/guide/storage/storageWithStore.adoc (hunk 1 above)
Comment (sdelamo) [reply src/main/docs/guide/storage/storageWithStore.adoc]: resolved via https://github.com/micronaut-projects/micronaut-microstream/pull/74/commits/1019bbc521b61fdf7033d91b9283c90780d006e4

File: src/main/docs/guide/storage/storageWithStore.adoc (hunk 2 above)
Comment (sdelamo) [reply src/main/docs/guide/storage/storageWithStore.adoc]: resolved via https://github.com/micronaut-projects/micronaut-microstream/pull/74/commits/1019bbc521b61fdf7033d91b9283c90780d006e4

**Review by timyates — APPROVED** [review]

**Pull request merged**